packages/cli-lib/lib/config.js

```diff
@@ -364,7 +364,7 @@ const isTrackingAllowed = () => {
 
 const getAndLoadConfigIfNeeded = (options = {}) => {
   if (!_config) {
-    loadConfig(null, {
```

Comment:
Are we sure this change will not cause a regression?

Reply:
I changed this back just in case -- I didn't see any regressions in my testing though.